src/App.css

```diff
@@ -0,0 +1,38 @@
+.App {
+  text-align: center;
+}
+
+.App-logo {
+  height: 40vmin;
+  pointer-events: none;
+}
+
+@media (prefers-reduced-motion: no-preference) {
+  .App-logo {
```

Comment:
Do you really need all this file? 
Are you using this styles for rotating the react logo?

Reply:
oh ... you do, but don't create PRs for boilerplate code. or code that was autogenerated 
for starting a new project usually:
- Create the project with autogenerated code
- Clean the unnecessary code
- push that code into main, develop or master branch (that's up to you)
- Create a new functionality on a different branch
- Open a PR for that functionality